packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,263 @@
+import { Pressable, StyleProp, Text, View, ViewStyle } from 'react-native'
+import { spacing } from '@department-of-veterans-affairs/mobile-tokens'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
+
+import { ComponentWrapper } from '../../wrapper'
+import { Icon } from '../Icon/Icon'
+import { Spacer } from '../Spacer/Spacer'
+import { useTheme } from '../../utils'
+
+export type CheckboxProps = {
+  /** Primary text for checkbox */
+  label: string
+  /** OnPress function. Pass a function that alters `checked` state */
+  onPress: () => void
+  /** Optional boolean that determines whether checkbox appears checked */
+  checked?: boolean
+  /** Optional description that appears below label */
+  description?: string
+  /** Optional accessibility override for description text  */
+  descriptionA11y?: string
+  /** Optional hint text. Appears below header */
+  hint?: string
+  /** Optional accessibility override for hint text  */
+  hintA11y?: string
+  /** Optional error text. Non-null value styles checkbox in error state */
+  error?: string
+  /** Optional accessibility override for error text  */
+  errorA11y?: string
+  /** Optional header text */
+  header?: string
+  /** Optional accessibility override for header  */
+  headerA11y?: string
+  /** Optional indeterminate boolean. Overrides checked styling */
+  indeterminate?: boolean
+  /** Optional accessibility override for label  */
+  labelA11y?: string
+  /** Optional required boolean that denotes a label as (*Required) */
+  required?: boolean
+  /** Optional tile styling */
+  tile?: boolean
+}
+
+export const Checkbox: FC<CheckboxProps> = ({
+  checked,
+  label,
+  labelA11y,
+  description,
+  descriptionA11y,
+  error,
+  errorA11y,
+  header,
+  headerA11y,
+  hint,
+  hintA11y,
+  indeterminate,
+  onPress,
+  required,
+  tile,
+}) => {
+  const { t } = useTranslation()
+  const theme = useTheme()
+
+  const baseTextStyle = {
+    fontFamily: 'SourceSansPro-Regular',
```

Comment:
Don't feel super strongly, but suggest reworking this slightly to have separate "fonts" so that the typography tokens can be dropped in more easily than having fontFamily split from fontSize/lineHeight and needing to shuffle around more later.